pandas/core/groupby/generic.py

```diff
@@ -926,7 +927,7 @@ def aggregate(self, func=None, *args, **kwargs):
 
     agg = aggregate
 
-    def _iterate_slices(self) -> Iterable[Tuple[Hashable, Series]]:
+    def _iterate_slices(self) -> Iterable[Tuple[Optional[Hashable], Series]]:
```

Comment:
note there is a similar definition for SeriesGroupBy - not sure if that needs to change as well

Reply:
maybe IndexLabel = Optional[Hashable] in _typing.py